packages/access-api/src/bindings.d.ts

```diff
@@ -59,6 +60,7 @@ export interface RouteContext {
     spaces: Spaces
     validations: Validations
     accounts: Accounts
+    delegations: DelegationsStorage
```

Comment:
Please change all or be consistent...🙏

```suggestion
    delegations: Delegations
```

Reply:
I changed the import name so it looks like the others.

The thing is, this isn't the same as the others. This is an interface, whereas the other 'models' here point to classes.
@Gozala and I discussed in this work trying eager to apply the [dependency inversion principle](https://en.wikipedia.org/wiki/Dependency_inversion_principle) and depend on an interface here instead of an implementation/class.

So that's why the name is inconsistent, it's not doing the same thing as others.

Reply:
OK gotcha - next time just add that info to the PR description so I know it's intentional!